ultravox/model/ultravox_config_test.py

```diff
@@ -0,0 +1,37 @@
+import pytest
+import transformers
+
+from ultravox.model import ultravox_config
+
+
+@pytest.mark.parametrize(
+    "model_id",
+    ["fixie-ai/ultravox-v0_2", "fixie-ai/ultravox-v0_3", "fixie-ai/ultravox-v0_4"],
+)
+def test_can_load_release(model_id: str):
```

Comment:
I think this could be a good way to make sure we stay backward compatible. If there's a breaking change needed, then we can consciously decide if we want to go ahead with it.

It obviously won't cover everything, but it should be a good start.